README.md

```diff
@@ -31,6 +31,110 @@ Deployment to `staging` and `production` servers is automated via [Travis-CI](ht
 
 `master` – Deploys to https://webmaker-desktop-production.herokuapp.com
 
+## Localization
+
+In this project we're using [React-Intl](https://github.com/yahoo/react-intl) to localize our application and YAML for translation.
+
+#### Localize a component or page
+
+To localize a component or page you have to include `IntlMixin` in your class `mixins`, for example:
+
+``` typescript
+var React = require('react');
+var IntlMixin = require('react-intl').IntlMixin;
+
+var Example = React.createClass({
+  mixins: [IntlMixin],
```

Comment:
our style is generally to require the mixin directly in the the array here, so like

```
mixins: [require('react-intl').IntlMixin]
```